### DISCUSSION TITLE
gh-pages branch is not generating

### DISCUSSION THREAD

**Discussion by vidvansh** [cotes2020/jekyll-theme-chirpy#1184] [Q&A] [answered]
I am trying to push changes to github but it is not creating any gh-pages branch and i think because of that any page i am pushing is not being shown on the website. I have tried everything. Can some help me fixing this issue.

**Comment by NichtsHsu** [comment, marked as answer]
Chirpy doesn't generate `gh-pages` branch, you should set the github pages source to Github Action:

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/35688135/3f1dd9d4-b495-4cd2-bc3a-6c6e743cacf8)

**Comment by ottohellwig** [comment]
@vidvansh I have seen from your repository that things appear to be configured correctly, if you are trying to make changes/update the website config/posts then you should refresh the page and wait for 1-2 minutes. A modal will appear saying there is new content and provide a button to update the website, click on it and your updates will be loaded.

In short:
- Push changes to GitHub repository
- Wait for sync then use the modal that appears
